Resources/doc/first_steps.rst

```diff
@@ -321,16 +320,16 @@ method of the document manager:
     $dm->remove($product);
     $dm->flush();
 
-As you might expect, the ``remove()`` method notifies Doctrine that you'd
-like to remove the given document from the MongoDB. The actual delete operation
-however, isn't actually executed until the ``flush()`` method is called.
+The ``remove()`` method notifies Doctrine that you'd like to remove
+the given document from the MongoDB. The actual delete operation
+however, isn't executed until the ``flush()`` method is called.
 
 Querying for Objects
 --------------------
 
 As you saw above, the built-in repository class allows you to query for one
-or many objects based on an number of different parameters. When this is
-enough, this is the easiest way to query for documents. Of course, you can
+or many objects based on any number of different parameters. When this is
+enough, this is the recommended way to query for documents. Of course, you can
```

Comment:
This is literally the easiest way, no need to avoid this word altogether. Also personally I'm far from recommending it :)

Reply:
Right, no need to overdo it in this case